ovos-i2csound

```diff
@@ -98,6 +98,10 @@ main() {
     if [[ ${detection_results[MARK1]} == true ]] ; then
         echo "Installing and configuring WM8731 based sound HAT"
         dtoverlay proto-codec
+        dtoverlay disable-bt
+        dtoverlay miniuart-bt
```

Comment:
miniuart-bt doesnt seem to be needed, or even do anything

i have tested this script and this doesnt solve it, if i edit config.txt directly it does work, but via ovos-i2csound i get errors

my test is simply
```
(.venv) ovos@raspOvos:~ $ echo "mouth.text=I am angry!" > /dev/ttyAMA0
-bash: /dev/ttyAMA0: Permission denied
```

if i add `dtoverlay=disable-bt` to config.txt then it works fine and i dont have Permission denied error

in both cases
```
$ dtoverlay -l
Overlays (in load order):
0:  proto-codec
1:  disable-bt
``

I do not know why it isn't working.... they should be equivalent but this is what I am observing

Reply:
See my comment below.